src/form.js

```diff
@@ -145,29 +145,49 @@ define([
     }
 
     Form.prototype = {
-        addHashtag: function(hashtag, xpath) {
-            this.hashtagDictionary[hashtag] = xpath;
+        isValidHashtag: function(tag) {
+            return this.hashtagDictionary.hasOwnProperty(this.normalizeHashtag(tag));
+        },
+        addHashtag: function(hashtag, xpath, dontOverwrite) {
```

Comment:
The databrowser module confusingly has a function with the same name and number of arguments, except the third argument is `vellum`. What do you think of making this a two-arg function `addHashtag(hashtag, xpath)`, which adds hashtag if it does not already exist, and adding a new function `overwriteHashtag(hashtag, xpath)`?

Also, reasoning about the default value of this `dontOverwrite` argument is confusing (it defaults to undefined, which means overwrite by default). Double negatives are hard.